src/emailpro/domain/emailpro-domain.html

```diff
@@ -170,15 +170,18 @@
                                         data-ng-click="updateDomain(domain)">
                                     <span class="oui-icon oui-icon-pen_line oui-icon_small" aria-hidden="true"></span>
                                 </button>
-                                <span data-uib-tooltip="{{ getDeleteTooltip(domain) }}"
+                                <button class="btn btn-icon"
+                                        type="button"
+                                        title="{{domain.displayName}}.&nbsp;{{::i18n.table_delete_entry}}"
+                                        data-ng-click="deleteDomain(domain)"
+                                        data-ng-if="!isDeleteDisabled(domain)">
+                                    <span class="oui-icon oui-icon-trash_line oui-icon_small" aria-hidden="true"></span>
+                                </button>
+                                <span class="btn btn-icon disabled"
```

Comment:
It's strange to put `btn btn-icon disabled`classes on a `span`. Can we find a better solution ?